cupy/cuda/cufft.pyx

```diff
@@ -327,7 +338,7 @@ class Plan1d(object):
         if fft_type != CUFFT_C2C and fft_type != CUFFT_Z2Z:
             raise ValueError('Currently for multiple GPUs only C2C and Z2Z are'
                              ' supported.')
-        if isinstance(devices, list):
+        if isinstance(devices, (tuple, list)):
```

Comment:
This could be `collections.abc.Iterable` to match the ValueError message below, but I think it is probably best to keep it as-is so we don't allow iterables like `cupy.ndarray`.